woodwork/utils.py

```diff
@@ -620,3 +622,15 @@ def _replace_nan_strings(df: pd.DataFrame) -> pd.DataFrame:
         df[col] = replaced_series
 
     return df
+
+
+def _parse_latlong(latlong):
+    nan_values = [x for x in ww.config.get_option("nan_values") if len(x)]
+    nan_values = "|".join(nan_values)
+
+    latlong = re.sub(nan_values, "None", latlong)
+
+    try:
+        return ast.literal_eval(latlong)
+    except ValueError:
+        pass
```

Comment:
should we be handling this ?

Reply:
This code was extracted from `_reformat_to_latlong`. I think the intent is to try and parse a string to return latlong if successful otherwise return None.